cypress/commons/actions/generic/ScenarioSelector.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// Select the scenario with the provided name and id
+import { apiUtils as api } from '../../utils';
+import { ScenarioParameters } from './ScenarioParameters';
+import { Scenarios } from './Scenarios';
+
+function selectScenario(scenarioName, scenarioId) {
```

Comment:
The previous function `selectScenario` in file `cypress/commons/actions/generic/Scenarios.js` still exists. If it is no longer used, could you remove it please ?